plugins/user_quota/girder_user_quota/web_client/views/QuotaPoliciesWidget.js

```diff
@@ -137,7 +137,7 @@ var QuotaPoliciesWidget = View.extend({
         sizeInfo = sizeToValueAndUnits(
             this.model.get('quotaPolicy').fileSizeQuota);
         defaultQuota = this.model.get('defaultQuota');
-        if (!defaultQuota) {
+        if (defaultQuota === null) {
             defaultQuotaString = 'Unlimited';
         } else {
             defaultQuotaString = formatSize(defaultQuota);
```

Comment:
The test failure message `ERROR: TypeError: undefined is not an object (evaluating 'sizeVal.toFixed')` makes me think `defaultQuota` might be `undefined` in the test.